delivery_ups_oca/README.rst

```diff
@@ -0,0 +1,107 @@
+================
+Delivery UPS OCA
+================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Production%2FStable-green.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Production/Stable
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fdelivery--carrier-lightgray.png?logo=github
+    :target: https://github.com/OCA/delivery-carrier/tree/13.0/delivery_ups_oca
+    :alt: OCA/delivery-carrier
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/delivery-carrier-13-0/delivery-carrier-13-0-delivery_ups_oca
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/99/13.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module adds `UPS <https://ups.com>`_ to the available carriers.
+
+It allows you to register shippings, generate labels, get rates from order, read shipping states and
+cancel shipments using UPS webservice, so no need of exchanging any kind of file.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+#. Add a carrier account with delivery type ``ups`` and fill in your credentials
+#. Configure the required fields of the UPS tab with your account data* https://wwwapps.ups.com
```

Comment:
Specify which fields and the URL seems incorrect. Another thing is that RST syntax doesn't allow `*` as footnotes. Use the proper one or don't put it as footnote (see my following suggestion).

I also miss something about the service code, and about packaging.

Reply:
Ok, I'm going to change the URL to https://wwwapps.ups.com/ppc/ppc.html
According to these comment https://github.com/OCA/delivery-carrier/pull/401#discussion_r684116111 I don't know where the fields to fill in are because I don't have a UPS account to verify it.

Reply:
I mean in Odoo, not in UPS. Mention the required fields, the "Track update..." field, etc.

Reply:
I have specified that it is in Odoo and that all the required fields in the UPS tab have to be filled in.